pilot/pkg/model/push_context.go

```diff
@@ -153,7 +153,7 @@ type XDSUpdater interface {
 	// This interface is WIP - labels, annotations and other changes to service may be
 	// updated to force a EDS and CDS recomputation and incremental push, as it doesn't affect
 	// LDS/RDS.
-	SvcUpdate(shard, hostname string, ports map[string]uint32, rports map[uint32]string)
+	SvcUpdate(shard, hostname string, namespace string, event Event, ports map[string]uint32, rports map[uint32]string)
```

Comment:
We donot need the ports and rports param

Reply:
I think may be better to leave them for future? Because they were there from the beginning.

Reply:
or if you are ok to clean it up - I can do that. I do not have strong opinion on that.

Reply:
Let's clean up it.

Reply:
Ok. Cleaned it up. PTAL